lisa/node.py

```diff
@@ -48,6 +48,7 @@ def __init__(
         self.is_default = runbook.is_default
         self.capability = runbook.capability
         self.name = runbook.name
+        self.is_test_target = True
```

Comment:
The default value should be False, and it can be a parameter of __init__ method. It's set to trye, when calling from local() and quick_connect().

Reply:
updated